File: TP/TP6/ex1.c
#include <stdio.h>
int main()
{

    char *days[8] =
        {
            "Day Doesn't exist",
            "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};

    int day;
    printf("Enter Day Number \n");
    scanf("%d", &day);
    if (day >= 1 && day <= 7)
    {
        printf("%s", days[day]);
    }
    else
    {
        printf("%s", days[0]);
    }
    return 0;
}

File: SEM2/PYTHON/TP4/ex1.py
days = [
    "monday", 
    "tuesday", 
    "wednesday", 
    "thursday", 
    "friday", 
    "saturday", 
    "sunday"
]


for day in days :
    print(day)
print("day f = ",days[3])

days[0],days[len(days)-1] = days[len(days)-1], days[0] 

print(days)
for i in range (12):
    print(days[len(days)-1])

